build.sbt

```diff
@@ -2,18 +2,18 @@ import bintray.Keys._
 
 name := "play-scalajs-scripts"
 
-version := "0.4.1-SNAPSHOT"
+version := "0.5.0-SNAPSHOT"
```

Comment:
Thanks for your PR! Could you just remove the `version` change please? I would like to do it later, in another commit.
Once the `build.sbt` file edited, you can execute the following git commands to update your branch without adding another commit:
```
git commit build.sbt --amend -C HEAD
git push --force origin play25
```